src/main.rs

```diff
@@ -15,7 +15,7 @@ use clap::{Args, Parser, Subcommand};
 use std::path::PathBuf;
 
 #[derive(Debug, Parser)]
-#[clap(name = "forc wallet", about = ABOUT, after_help = EXAMPLES, version)]
+#[clap(name = "forc wallet", about = ABOUT, after_long_help = EXAMPLES, version)]
```

Comment:
> subcommands/options first then scroll up to see the examples?

Interesting - I always start reading from the top of whatever output I'm given, so this would make things back to front for me :smiling_face_with_tear: I might leave this as is for now, but we can discuss in a followup issue?